source/val/validate_builtins.cpp

```diff
@@ -373,6 +373,13 @@ class BuiltInsValidator {
   spv_result_t ValidateMeshShadingEXTBuiltinsAtDefinition(
       const Decoration& decoration, const Instruction& inst);
 
+  // Used as a common method for validating MeshEXT builtins
+  spv_result_t ValidateMeshEXTBuiltinInterfaceRules(
+      const Decoration& decoration, const Instruction& inst,
+      spv::Op scalar_type, uint32_t array_type_vuid, uint32_t array_size_vuid,
```

Comment:
instead of passing in 5 VUID ints, seems it would be easier to pass in the `spv::Builtin` and then just set them inside this function in a single spot

Reply:
Can you show an example?

Reply:
> Can you show an example?

look at `GetVUIDForBuiltin`

now try to re-write it where EVERY function is just randomly passing in random VUID ints around

Reply:
how about i make a local static array inside ValidateMeshEXTBuiltinInterfaceRules?